### ISSUE TITLE
Add Other Search Engines #Hacktoberfest

### ISSUE THREAD

**Issue by TheSpeedX** [TheSpeedX/TorrGrab#3]
This Current Module Currently Scrapes Off Data From PirateBay

What You Can do?

[ - ] Add Other Torrent Search Engine Scrapper
    Make Sure That The User Can Access it Without VPN
[ - ] Make The Torrent File Downloadable On Command Line Without 3rd Party Clients

**Comment by Iamshankhadeep** [comment]
Can I work on it?

**Comment by TheSpeedX** [comment]
@Iamshankhadeep Yes You can work and do the following improvements and add something new too

**Comment by robbyq92** [comment]
maybe for this
https://divxtotal7.com/
have magnet
